pandas/conftest.py

```diff
@@ -8,9 +8,10 @@
 from hypothesis import strategies as st
 import numpy as np
 import pytest
-from pytz import FixedOffset, utc
+from pytz import FixedOffset, utc  # type: ignore
```

Comment:
This is the same as #28932 - I think this needs to be fixed upstream in Typeshed. Would you be interested in submitting that there?

Reply:
Sure - I can give it a shot.

Reply:
Picking this up again - just got the changes merged in typeshed (https://github.com/python/typeshed/pull/3393). I see that there are now some conflicts with master; I can address those and pick this up again in the next couple days.